### ISSUE TITLE
SA1009 (ClosingParenthesisMustBeSpacedCorrectly) reported when preceded by comment

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#684]
The following statement triggers SA1009 on the closing `)` character.

```csharp
return new MutationResult(newRoot, 1 /*result == OperationResult.SizeChanged ? 1 : 0*/);
```

It also reports SA1009 if the space between the `1` and the comment is removed:

```csharp
return new MutationResult(newRoot, 1/*result == OperationResult.SizeChanged ? 1 : 0*/);
```

I am not yet sure how we want to handle this.

**Comment by Przemyslaw-W** [comment]
IMHO SA1009 should not be reported in any of above code as there is no
space before closing ).
I think there should be additional spacing rules for /* and */. The rule
for /* should report missing space in the second case

2015-04-23 4:08 GMT+02:00 Sam Harwell <notifications@github.com>:
